README.md

```diff
@@ -1,52 +1,85 @@
-# Data Diff
+# **data-diff**
 
-**`data-diff` is currently under heavy development, if you run into issues,
+**data-diff is currently under heavy development, if you run into issues,
 please file an issue and we'll help you out ASAP!**
 
-A cross-database, efficient diff using checksums between mostly-similar database
-tables.
-
-- Validate that a table was copied properly
-- Be alerted before your customer finds out, or your report is wrong
-- Validate that your replication mechnism is working correctly
-- Find changes between two versions of the same table
-
-It uses a bisection algorithm and checksums to efficiently check if e.g. a table
-is the same between MySQL and Postgres, or Postgres and Snowflake, or MySQL and
-RDS!
-
-```python
-$ data-diff postgres:/// Original  postgres:/// Original_1diff -t timestamp -v --bisection-factor=4
-[16:57:36] INFO - Diffing tables of size 25000095 and 25000095 | segments: 4, bisection threshold: 1048576.
-[16:58:03] INFO - Diffing segment 1/4 of size 8333364 and 8333364
-[16:58:12] INFO - Diffing segment 2/4 of size 8333365 and 8333365
-[16:58:29] INFO - . Diffing segment 1/4 of size 2777787 and 2777787
-[16:58:32] INFO - . Diffing segment 2/4 of size 2777788 and 2777788
-[16:58:45] INFO - . . Diffing segment 1/4 of size 925925 and 925925
-[16:58:46] INFO - . . Diffing segment 2/4 of size 925929 and 925929
-[16:58:48] INFO - . . . Diff found 2 different rows.
-+ (12500048, 1268104625)
-- (12500048, 1268104626)
-[16:58:48] INFO - . . Diffing segment 3/4 of size 925929 and 925929
-[16:58:49] INFO - . . Diffing segment 4/4 of size 5 and 5
-[16:58:50] INFO - . Diffing segment 3/4 of size 2777788 and 2777788
-[16:58:52] INFO - . Diffing segment 4/4 of size 2 and 2
-[16:58:55] INFO - Diffing segment 3/4 of size 8333365 and 8333365
-[16:59:00] INFO - Diffing segment 4/4 of size 1 and 1
-[16:59:00] INFO - Duration: 89.92 seconds.
-```
-
-We currently support the following databases:
-
-- PostgreSQL
-- MySQL
-- Oracle
-- Snowflake
-- BigQuery
-- Redshift
-
-We plan to add more, including NoSQL, and even APIs like Shopify! We're very
-open to contributions on this front.
+**data-diff** is a command-line tool and Python library to efficiently validate
+whether two different databases are in sync, e.g. Postgres and Redshift. The
+common use-case is to verify replication between two datastores with e.g.
+Fivetran, Debezium, .. and fix missing rows and/or alert on missing/incorrect
+data so you know whether something is wrong before your customer.
+
+**data-diff** doesn't pull down and compare every row, because it is extremely
```

Comment:
It kinda sounds like you are calling data-diff slow, lol.

```suggestion
**data-diff** doesn't pull down and compare every row, because that would be extremely
```